src/pf-notification-drawer/pf-notification-drawer-accordion.component.js

```diff
@@ -0,0 +1,51 @@
+import {pfUtil} from 'pf-utils.js';
+
+/*
+ * <b>&lt;pf-notification-drawer-accordion&gt;</b> element for Patternfly Web Components
+ *
+ * @example {@lang xml}
+ * <pf-notification-drawer-accordion>
+ *   <pf-notification-drawer-accordion-panel></pf-notification-drawer-accordion-panel>
+ *   <pf-notification-drawer-accordion-panel></pf-notification-drawer-accordion-panel>
+ *   . . .
+ * </pf-notification-drawer-accordion>
+ */
+
+export class PfNotificationDrawerAccordion extends HTMLElement {
+
+  /*
+   * An instance of the element is created or upgraded
+   */
+  constructor () {
+    super();
+  }
+
+  /*
+   * Called every time the element is inserted into the DOM
+   */
+  connectedCallback () {
+    let that = this;
+    pfUtil.addClass(this, 'panel-group');
+
+    let observer = new MutationObserver(function(mutations) {
```

Comment:
@dlabrecq after observing the performance a bit more, I haven't noticed a huge impact. As @agathver was reporting before, a lot of Polymer still uses this pattern and the impact is minimal. In cases where transclusion is necessary (i.e. events/methods aren't as useful), I still see the MutationObserver as a viable option. This also helps us internally abstract the issue and minimize the impact later on when we adopt Shadow DOM. I.m.o. it will be much easier from an upgrade standpoint then. I should also add that I have heard many React/Angular developers recently discussing the addition of Shadow DOM into there solutions (so it'd be great if we can get going on this once it solidifies).

Another related topic, I am also now attempting to minimize DOM transclusion where possible (i.e. if i don't need to copy the nested DOM into a new template, I just let it render inside the CE, there's no need to create templates unnecessarily). I've proposed that DOM elements which are semantic `button`, `input`, etc. be left to the consumer (i.e. like our `pf-dropdown` does) and that non-semantic `divs` be encapsulated into CE where possible. This accomplishes three goals:
* simplifies the DOM structure / avoids unnecessary transclusion
* allow the outside framework to still add bindings/change inner DOM (if it's not transcluded)
* allows the CE to interact w/ the innerDOM and just provide minimal behaviors/enhance accessbility/ensure common styles/ etc.

Deciding which approach is easiest (MutationObserver, events/method updates, inner DOM) seems to just be a case by case analysis of what the consumer is comfortable with and is consistent. We'd definitely appreciate anymore feedback on this from an Angular perspective and I'll be trying to do some more Angular demos soon...